api/pkg/apis/projectcalico/v3/bgppeer.go

```diff
@@ -100,6 +100,10 @@ type BGPPeerSpec struct {
 	// This removes BGP loop prevention and should only be used if absolutely necesssary.
 	// +optional
 	NumAllowedLocalASNumbers *int32 `json:"numAllowedLocalASNumbers,omitempty"`
+	// TTLSecurity enables the generalized TTL security mechanism (GTSM). The provided value is the number of hops
+	// (edges) between the peers.
```

Comment:
Probably worth adding a bit more context to this.

e.g.,

"GTSM protects against spoofed packets by ignoring received packets with a smaller than expected TTL"

Reply:
Done.